### ISSUE TITLE
Remove all menubar css and js

### ISSUE THREAD

**Issue by colemanw** [civicrm/org.civicrm.shoreditch#342]
The soon-to-be-in-core [KAM extension](https://github.com/aydun/uk.squiffle.kam) natively implements basically all the things that Shoreditch was adding onto the menubar - lighter color scheme, robust markup, larger size, etc.

I'd recommend we remove *all* menubar-related css and JS from Shoreditch and start from scratch if there are any remaining tweaks to be done.

**Comment by monishdeb** [comment]
I agree. I faced an issue with q-search icon which is specific to shoreditch custom css changes, after KAM extension is installed:
<img width="860" alt="screen shot 2018-12-12 at 2 29 30 pm" src="https://user-images.githubusercontent.com/3735621/49858837-0454b600-fe1c-11e8-971c-8139901d8db7.png">

I fixed it by removing the css styling for q-search, see [here](https://github.com/JMAConsulting/org.civicrm.shoreditch/commit/f0df1e6df98dbc97bb8c3b767f4660faa72b8d1a)

After fix:
<img width="830" alt="screen shot 2018-12-12 at 2 43 06 pm" src="https://user-images.githubusercontent.com/3735621/49858955-539ae680-fe1c-11e8-9d2f-264caf5efc13.png">

**Comment by AkA84** [comment]
@colemanw do you know, approximately, when the KAM extension will be merged into core? It would be best to remove anything menu-related (or to heavily amend it at least) only after the extension has become part of core

**Comment by colemanw** [comment]
@AkA84 probably next month, which would set the version at 5.11.
But cleaning up the css/js in this extension and testing it with KAM could happen now. It could even be released now, with KAM declared as a dependency. Once KAM is merged with core that dependency would be automatically resolved, based on some work I'm doing right now on the extension management system.

**Comment by colemanw** [comment]
@jamienovick @AkA84 The inclusion of KAM in core is looming closer with https://github.com/aydun/uk.squiffle.kam/issues/16 wrapping up. I strongly suggest you get someone to test out that extension with Shoreditch in the next week or so in case there is something critical missing from KAM you'd like to see. Pushing changes into that extension is fairly easy while it remains an extension...

**Comment by AkA84** [comment]
@colemanw thanks for the heads-up! The ~1w window though is a bit unfortunate given the proximity to the holidays, so I'm not sure that all the necessary (extensive testing, tweaking of the theme to properly add its own look-and-feel to the menu, etc) can be done before I'm back. For consideration: this could be attacked as part of the community efforts (see https://lab.civicrm.org/extensions/shoreditch/issues/1)

(cc @jamienovick )

**Comment by colemanw** [comment]
@AkA84 @jamienovick the holidays slowed me down as well so this is getting pushed to 5.12. However several extensions including CiviTutorial and RecentMenu now declare KAM as a dependency (when KAM is merged into core that dependency will automatically be resolved). I think the time is now for ShoreDitch to do the same.

**Comment by colemanw** [comment]
https://github.com/civicrm/civicrm-core/pull/13582 has been opened, targeting 5.12 for inclusion in core.

**Comment by eileenmcnaughton** [comment]
I just had this reported to me as an issue by a user after we enabled KAM

**Comment by eileenmcnaughton** [comment]
I have the 5.13 rc on our staging & locally & it just started to look really bad - I'm gonna try the patch above

![Screen Shot 2019-04-13 at 1 29 49 PM](https://user-images.githubusercontent.com/336308/56074122-6f28f480-5df0-11e9-9027-ec768dcea0ec.png)

**Comment by twomice** [comment]
FYI, the display issue mentioned by @eileenmcnaughton [above](https://github.com/civicrm/org.civicrm.shoreditch/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+menu#issuecomment-482772426) is resolved by removing the background-color rule for #civicrm-menu ([see here](https://github.com/twomice/org.civicrm.shoreditch/commit/7257f92d48c72aa3c278af4ca2c31142f2a04618#diff-cfa3de0423d55cd3a3636933b96af457L177))

**Comment by eileenmcnaughton** [comment]
Cool - I've been adding css through a hook to override shoreditch where it makes me sad - maybe I should include that

**Comment by themak1985** [comment]
@twomice how are you implementing this with alpha31?

**Comment by twomice** [comment]
@themak1985 I haven't had to face that yet.

**Comment by jamienovick** [comment]
Closing as fixed in latest version
